lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/graphing-models.adoc

```diff
@@ -0,0 +1,87 @@
+= Graphing Quadratic Models
+++++
+<style>
+.autonum { font-weight: bold; }
+.autonum:after { content: ')' !important; }
+
+.FillVerticalSpace { grid-gap: 20px; margin-top: 5px; }
+
+td {
+	background-color: rgba(#fff, .5);
+	background-image:
+		linear-gradient(lightblue 2px, transparent 2px),
+		linear-gradient(270deg, lightblue 2px, transparent 2px),
+		linear-gradient(rgba(150, 150, 150, .3) 1px, transparent 1px),
+		linear-gradient(270deg, rgba(150, 150, 150, .3) 1px, transparent 1px);
+	background-size: 100px 100px, 100px 100px, 20px 20px, 20px 20px;
+	background-position: -3px 74px;
+	min-height: 1.5in;
+	max-width: 2in;
+}
+
+td p.tableblock { position: absolute; }
+
+/* Add labels for axes */
+td::before, td::after { display: flex; font-style: italic; }
+td::before {
+	content: 'mpg';
+	justify-content: center;
+	align-items: center;
+	margin-right: 1.5em;
+	margin-top: -2em;
+    -webkit-transform:rotate(270deg);
+    -moz-transform:rotate(270deg);
+    -o-transform: rotate(270deg);
+    -ms-transform:rotate(270deg);
+    transform: rotate(270deg);
+}
+td::after {
+	content: 'speed';
+	align-items: start;
+	justify-content: right;
+	margin-top: 2em;
+	margin-right: -0.5em;
+}
+
+</style>
+++++
+
+For this page, you'll need to have @starter-file{alg2-exploring-quadratics} open in Desmos on your computer. The parabola you see is the graph of @math{f(x)=x^2}. Another, *identical* parabola is hiding behind it. This second parabola is written in Vertex Form: @math{g(x)=a(x-h)^2+k}. Each coefficient starts at values to make @math{g(x)} equivalent to @math{f(x)}.
+
+@n Use the values of @math{a}, @math{h}, and @math{k} from Desmos, write the Vertex Form of @math{f(x) = x^2}: @math{f(x)=}@fitb{}{@ifsoln{@math{1(x-0)^2+0}}}
+
+== Magnitude @math{a}
+@n Try changing the value of @math{a} to -4, 0, and 2, graphing each parabola in the squares below and labeling the vertex and roots.
+
```

Comment:
- Please clarify whether you want students to label vertex and roots with words or coordinates?
- Label not labeling
- Would it serve your purpose to provide space beneath each graph to record these values?